src/style.scss

```diff
@@ -0,0 +1,119 @@
+@import "../node_modules/bootstrap/scss/bootstrap";
+@import "../node_modules/@riophae/vue-treeselect/dist/vue-treeselect";
+
+.dynamic-form {
+
+  input[type=number] {
+    max-width: 120px;
+  }
+}
+
+.icon-small svg {
+  height: 20px;
+  width: 20px;
+  vertical-align: sub;
+}
+
+.content {
+
+  .vue-treeselect {
+
+    .vue-treeselect__control {
+      @extend .form-control;
+      font-size: inherit;
+      display: table;
+      padding-bottom: 0;
+    }
+
+    &:not(.vue-treeselect--disabled) {
+      .vue-treeselect__placeholder {
+        color: inherit;
+      }
+    }
+  }
+
+  .vue-treeselect--focused.vue-treeselect--open, .vue-treeselect--focused:not(.vue-treeselect--open) {
+    .vue-treeselect__control {
+      @extend .form-control:focus;
+      display: table;
+      padding-bottom: 0;
+    }
+  }
+
+  .vue-treeselect__multi-value-item-container {
+    padding-top: 0;
+    padding-bottom: 2px;
+  }
+
+  .vue-treeselect__single-value, .vue-treeselect__placeholder {
+    padding-left: 0;
+    line-height: inherit;
+    font-size: inherit;
+  }
+
+  .vue-treeselect--focused.vue-treeselect--searchable .vue-treeselect__single-value {
+    color: inherit;
+  }
+
+  .vue-treeselect__control-arrow-container {
+    padding-top: 8px;
+    vertical-align: top;
+  }
+
+  .vue-treeselect__control-arrow {
+    color: inherit;
+  }
+
+  form.was-validated {
+
+    .vue-treeselect__value-container {
+      display: block;
+    }
+
+    .vue-treeselect__control-arrow-container {
+      display: none;
+    }
+
+    .is-valid {
+      .vue-treeselect--focused.vue-treeselect--open, .vue-treeselect--focused:not(.vue-treeselect--open) {
+        .vue-treeselect__control {
+          @extend .form-control.is-valid:focus
+        }
+      }
+
+      .vue-treeselect__control {
+        @extend .form-control.is-valid
+      }
+    }
+
+    .is-invalid {
+      .vue-treeselect--focused.vue-treeselect--open, .vue-treeselect--focused:not(.vue-treeselect--open) {
+        .vue-treeselect__control {
+          @extend .form-control.is-invalid:focus
+        }
+      }
+
+      .vue-treeselect__control {
+        @extend .form-control.is-invalid
+      }
+
+      + .invalid-feedback {
+        display: block;
+      }
+    }
+  }
+
+  select {
+    background-image: linear-gradient(45deg, transparent 50%, $body-color 60%), linear-gradient(135deg, $body-color 40%, transparent 50%);
+    background-position: calc(100% - 15px) 15px, calc(100% - 9px) 15px, 100% 0;
+    background-size: 6px 6px, 6px 6px;
+    background-repeat: no-repeat;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+  }
+}
+
+.btn-submit {
```

Comment:
This is new; in hint we simply gave the submit button the class "btn-red", which is a hint specific class, so here I've given it the class "btn-submit" which defaults to bootstrap's "btn-success" styling but can easily be overriden in any parent project.